helpdesk/views/staff.py

```diff
@@ -273,8 +274,7 @@ def followup_edit(request, ticket_id, followup_id):
             'time_spent': format_time_spent(followup.time_spent),
         })
 
-        ticketcc_string, __ = \
-            return_ticketccstring_and_show_subscribe(request.user, ticket)
+        ticketcc_string = return_ticketccstring_and_show_subscribe(request.user, ticket)[0]
```

Comment:
You are right, this will be better with a unit test for this case. But my change shouldn't break anything since I get first element of the tuple with `[0]`. I wanted to get rid of the useless variable `__`

Reply:
Ah! ... I missed that [0]

Reply:
NOTE: If you use a single undescore instead of a double underscore, the linters will not complain about an unused variable.

Reply:
I missed the array indexing too, thanks. sounds good. do we need to update to single underscore to pass tests?